documentation/observability/5-deploy-to-kyma.md

```diff
@@ -4,34 +4,34 @@
 
 To start streaming logs, metrics and traces from SAP BTP, Kyma runtime to the SAP Cloud Logging service, you need to follow these steps:
 
-1. Enable the Kyma [Telemetry module](https://kyma-project.io/#/telemetry-manager/user/README) by following the steps described as [Quick Install of kyma-project.io](https://kyma-project.io/#/02-get-started/01-quick-install).
+1. Add the Kyma [Telemetry module](https://help.sap.com/docs/btp/sap-business-technology-platform/kyma-telemetry-module) by following the steps described in [Add and Delete a Kyma Module Using Kyma Dashboard](https://help.sap.com/docs/btp/sap-business-technology-platform/enable-and-disable-kyma-module).
 
-2. An instance of SAP Cloud Logging with OpenTelemetry is enabled to ingest distributed traces.
+2. An instance of SAP Cloud Logging with OpenTelemetry is enabled to ingest distributed traces and metrics.
 
 > [!TIP]
 >We recommend that you create an instance of SAP Cloud Logging with the SAP BTP service operator, because it takes care of creation and rotation of the required secret. See [Create an SAP Cloud Logging Instance through SAP BTP Service Operator](https://help.sap.com/docs/cloud-logging/cloud-logging/create-sap-cloud-logging-instance-through-sap-btp-service-operator)
-However, you can choose any other method of creating the instance and the secret, as long as the parameter for the OTLP ingestion is enabled in the instance. For details, see [Configuration Parameters](https://help.sap.com/docs/cloud-logging/cloud-logging/configuration-parameters?locale=en-US&version=Cloud).
+However, you can choose any other method of creating the instance and the secret, as long as the parameter for the OTLP ingestion is enabled in the instance. For details, see [Configuration Parameters](https://help.sap.com/docs/cloud-logging/cloud-logging/configuration-parameters).
 
 To create the SAP Cloud Logging service instance, follow these steps:
 
 1. Log in to the Kyma cluster.
 
 > [!TIP]
-> To log in to the Kyma cluster, follow the [Log In to Your Kyma cluster](https://developers.sap.com/tutorials/deploy-to-kyma.html#1206fdc5-e6d4-4238-8cef-638cc7739ca6) tutorial.
+> To log in to the Kyma cluster, follow the [Access a Kyma Instance Using kubectl](https://help.sap.com/docs/btp/sap-business-technology-platform/access-kyma-instance-using-kubectl) instructions.
 
 1. Create a namespace with the following command:
    
     ``` 
     kubectl create namespace sap-cloud-logging-integration 
     ```
 
-2. Create a new file called `cls.yaml` and paste the following code:
+2. Create a new file called `cloud-logging-instance.yaml` and paste the following code:
```

Comment:
Numbering is rendered weirdly here: 1, 2, 1, 1, 2, 3...

<img width="103" alt="image" src="https://github.com/user-attachments/assets/10cb84d6-1a11-4927-b472-f8bec42af904">

Reply:
fixed